### PR TITLE
[SPRINT-02] - [001] - Property name unification.

### PR DESCRIPTION
**Pull request by carlosfigueroa97**

### CODE REVIEW

File: src/helpers/properties.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    }
+}
+
+const StatusTypeClient = {
+    type: String,
+    required: true,
+    validate(value){
+        var type = Strings.StatusTypeClient
+        if(!type.getValue(value)){
+            throw new Error('Tipo de usuario inválido')
```
Comment (15mgm15) [inline src/helpers/properties.js]: Aqui veo que se repiten strings "Tipo de usuario inválido" se puede reutilizar?

File: src/helpers/strings.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const Enum = require('enum')
+
+var StatusType = new Enum({ 1: 'pending', 2: 'active', 3: 'inactive', 4: 'banned' })
+var StatusTypeClient = new Enum({ 1: 'unknown', 2: 'active', 3: 'inactive', 4: 'banned' })
```
Comment (15mgm15) [inline src/helpers/strings.js]: "active", "inactive", "banned" se pueden reutilizar?

File: src/helpers/properties.js (hunk 1 above)
Comment (carlosfigueroa97) [reply src/helpers/properties.js]: Si, esas strings las va a cambiar @PhonyFish no?

Comment (15mgm15) [reply src/helpers/properties.js]: Si es el caso, entonces este comentario no tiene sentido :D

File: src/helpers/strings.js (hunk 2 above)
Comment (carlosfigueroa97) [reply src/helpers/strings.js]: Yes, that's it. I'll fix it now.

**Comment by 15mgm15** [comment]
Please fix the conflicts

File: src/helpers/strings.js [hunk 3]
```diff
@@ -1,19 +1,27 @@
...
+err_day = 'Día inválido'
+err_Name_empty = 'Por favor ingrese el nombre'
+err_Code_empty = 'Por favor ingrese el código'
+err_Username_empty = 'Por favor ingrese el usuario'
+err_Password = 'Contraseña inválida'
+err_Password_empty = 'Por favor ingrese su contraseña'
+err_Password_length = 'La contraseña debe contener 8 dígitos como mínimo'
+err_Cellphone = 'Número de teléfono invalido'
+err_Cellphone_empty = 'Por favor ingrese el número de teléfono'
+err_TypeDeliveryman = 'El tipo de repartidor es inválido'
```
Comment (Colima-97) [inline src/helpers/strings.js]: I get the idea guys, but why are we using capital and non-capital letters for variables without any apparent order? 

Why do we have `"err_Mayus"` but then, we have `err_day`?

File: src/helpers/enums.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+const Enum = require('enum')
+
+const active = 'active'
+const inactive = 'inactive'
+const banned = 'banned'
+
+var StatusType = new Enum({ 1: 'pending', 2: active, 3: inactive, 4: banned })
+var StatusTypeClient = new Enum({ 1: 'unknown', 2: active, 3: inactive, 4: banned })
+var UserType = new Enum({ 1: 'client', 2: 'deliveryman', 3: 'restaurantadmin', 4: 'restaurantemployee', 5: 'thirdparty' })
+var DeliverymanType = new Enum({ 1: 'fixed', 2:'nonfixed' })
```
Comment (Colima-97) [inline src/helpers/enums.js]: Why do we have constants for `active`, `inactive` and `banned` but then, we don't have constants for `pending`, `unknown`, `client`, `deliveryman`, `fixed`, `non fixed`, etc.?

I mean, it's a good idea, but we should code all of them.

Comment (Colima-97) [reply src/helpers/enums.js]: @carlosfigueroa97 maybe you could write the constants into @PhonyFish strings' file

Comment (carlosfigueroa97) [reply src/helpers/enums.js]: Because the others do not repeat

Comment (carlosfigueroa97) [reply src/helpers/enums.js]: @Colima-97 Excellent idea

**Pull request merged**